packages/hiro-graph-client/src/index.js

```diff
@@ -15,4 +18,9 @@ export {
     appsServletFactory,
     kiServletFactory,
     variablesServletFactory,
+    // Lucene
```

Comment:
Do we need it?

Reply:
No harm in having it, it's easier than writing manually.
Will make another PR to use them automatically if an object is passed in to client.gremlin() or client.lucene(). Make sense?